SEED-platform/seed
# Nyc migration issue dec 2016

#### Any background context you want to provide?
    This pull request is targeted for an issue in NYC migrations but may involve other organizations as well, unknown at this point.  In a few cases there was some missing data causing a database call in create_m2m_relationships_organization.py to fail skipping any subsequent records.
#### What's this PR do?
    Wraps the database call in a try/except block, logs any errors and continues processing.  Also added more logging around various aspects of the migration code.
#### How should this be manually tested?
    Remigrate NYC data and check the known bad cases.  Also check a couple of the known good cases just to make sure they're still fine.
#### What are the relevant tickets?
  None.  There are google docs describing the issue but they contain propritary information.
#### Screenshots (if appropriate)
#### Definition of Done:
   I'm not sure how I would do this.
   No, I don't thing so.
   No, I don't think so.  This is fix for data migration and I don't think we are doing any regression testing for them.   Maybe I am just not seeing them?
    No
